<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KSelectDatabaseDlg</class>
 <widget class="QWidget" name="KSelectDatabaseDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>451</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>518</width>
    <height>679</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KMyMoney - Select Database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox6">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>111</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>126</red>
           <green>125</green>
           <blue>124</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="title">
      <string>PLEASE NOTE</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="WindowText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>126</red>
              <green>125</green>
              <blue>124</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>126</red>
              <green>125</green>
              <blue>124</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>This software is still in an experimental stage, and there are still some known response time problems. Please be patient; we are working on them.
As always, please make sure you have adequate backups of your data.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Database Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="databaseTypeCombo"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Fi&amp;le</string>
       </property>
       <property name="buddy">
        <cstring>urlSqlite</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KUrlRequester" name="urlSqlite">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QWidget" name="databaseFields" native="true">
       <layout class="QFormLayout" name="formLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Database &amp;Name</string>
          </property>
          <property name="buddy">
           <cstring>textDbName</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KLineEdit" name="textDbName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Host &amp;Name</string>
          </property>
          <property name="buddy">
           <cstring>textHostName</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="KLineEdit" name="textHostName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&amp;Username</string>
          </property>
          <property name="buddy">
           <cstring>textUserName</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="KLineEdit" name="textUserName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>&amp;Password</string>
          </property>
          <property name="buddy">
           <cstring>textPassword</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="KLineEdit" name="textPassword">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="checkPreLoad">
     <property name="text">
      <string>Preload &amp;all data</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>plainTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
